### PR TITLE
add instructor guide for terminal commands

### PR DESCRIPTION
**Pull request by mcritchlow**
Closes: #53 

Here's a first pass at a script that primarily focuses on the terminal commands. I "think" I have the latest commands represented here including the new addition for Conflicts. This is still a little rough, but I wanted to gauge whether this falls somewhat along the lines of what you all are thinking and would find useful.

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -3,293 +3,548 @@ layout: page
...
 
 ## [Setting Up Git]({{ page.root }}/02-setup/)
 
-*   We suggest instructors and students use `nano` as the text editor for this
-    lessons because
-
-    *   it runs in all three major operating systems,
-    *   it runs inside the shell (switching windows can be confusing to students), and
-    *   it has shortcut help at the bottom of the window.
+Use the setup page here.
```
Comment (chrissyrissmeyer) [inline _extras/guide.md]: these were added to the slide deck as well

File: _extras/guide.md [hunk 2]
```diff
@@ -3,293 +3,548 @@ layout: page
...
+* There are other patterns you can use to ignore files and folders.
+
+## [Working with Branches]({{ page.root }}/06-branching/)
+### Section Overview:
+- "What are branches and why should I use them?"
+- "How do I merge one branch into another?"
+
+### Instructions:
+
+- Intro and context setting
```
Comment (chrissyrissmeyer) [inline _extras/guide.md]: add: Use slides

**Review by chrissyrissmeyer — COMMENTED** [review]
Matt this all looks great to me! And it will be totally useful. It is a nice balance of context and commands.

Didn't review all the sections in depth -- just those I am doing (where I added a couple of comments).

File: _extras/guide.md [hunk 3]
```diff
@@ -3,293 +3,548 @@ layout: page
...
 
-    Please point out to students during setup that they can and should use
-    another text editor if they're already familiar with it.
-
-*   When setting up Git, be very clear what learners have to enter: it is
-    common for them to edit the instructor's details (e.g. email).  Check at
-    the end using `git config --list`.
+### Key Points
+* "Use `git config` to configure a user name, email address, editor, and other preferences once per machine."
+* "Use `git help` or `git help <command>` if you get stuck!
```
Comment (dunn) [inline _extras/guide.md]: Missing quotation mark at the end.

File: _extras/guide.md [hunk 4]
```diff
@@ -3,293 +3,548 @@ layout: page
...
+- **New Command**: `git init` to initialize our git repository
+~~~
+$ git init
+~~~
+{: .bash}
+
+- If we list the files, we can not see a hidden `.git` directory. Git stores all information about our project in here.
+  If we ever delete it, we will lose our project history
+~~~
+$ ls -a
```
Comment (dunn) [inline _extras/guide.md]: Do you want to add `dir /ah` for Windows?

File: _extras/guide.md [hunk 5]
```diff
@@ -3,293 +3,548 @@ layout: page
...
+- **New Command**: `git pull` to pull changes from our Github repository to our local repository. Pulling now has no
+  effect.
+~~~
+$ git pull origin master
+~~~
+{: .bash}
+
+### Key Points
+* "A local Git repository can be connected to one or more remote repositories."
+* "Use the HTTPS protocol to connect to remote repositories until you have learned how to set up SSH."
```
Comment (dunn) [inline _extras/guide.md:426]: I think HTTPS is recommended over SSH anyway.

**Comment by dunn** [comment]
So is the idea that we'll have this in front of us instead of the episodes themselves?  Or that we'll switch back and forth as we need more and less detail?

**Comment by mcritchlow** [comment]
I think the idea is this will mostly be the "script" while you're running through the terminal commands in an episode. Then if you need to switch to describing an example image on the slides/episodes you can do that. So if you want pointers in particular places to help you remember that let me know and I'll happily put them in.

File: _extras/guide.md (hunk 5 above)
Comment (mcritchlow) [reply _extras/guide.md:426]: Really? Interesting.

File: _extras/guide.md (hunk 4 above)
Comment (mcritchlow) [reply _extras/guide.md]: If they're using the Git Bash thing for Windows, do they need `dir` still, or do they have `ls`?

File: _extras/guide.md (hunk 3 above)
Comment (mcritchlow) [reply _extras/guide.md]: ty!

File: _extras/guide.md (hunk 1 above)
Comment (mcritchlow) [reply _extras/guide.md]: Ok I'll refer to the deck instead then.

File: _extras/guide.md (hunk 2 above)
Comment (mcritchlow) [reply _extras/guide.md]: will do!

**Comment by mcritchlow** [comment]
I think I've covered the change requests so far. Let me know if there are others.

File: _extras/guide.md (hunk 5 above)
Comment (dunn) [reply _extras/guide.md:426]: At least by GitHub, maybe just because it's more reliable from behind proxies: https://stackoverflow.com/a/11041782

**Review by dunn — APPROVED** [review]

**Pull request merged**